semantics.json

```diff
@@ -905,6 +905,13 @@
         "label": "Text for screenreaders. Title for zoom out button",
         "importance": "low",
         "default": "Zoom out"
+      },
+      {
+        "name": "zoomToolbar",
+        "type": "text",
+        "label": "Zoom toolbar",
+        "importance": "low",
+        "default": "Zoom toolbar"
```

Comment:
This will need to go into the `sanitizeContentTypeParameters` function in `utils/sanitization.js`, otherwise existing content will announce "undefined" unless it's edited and saved again.

Reply:
Yes, thanks for noticing!